api/contrib/envoy/extensions/network/connection_balance/dlb/v3alpha/dlb.proto

```diff
@@ -30,4 +30,8 @@ message Dlb {
   // The ID of the Dlb hardware, start from 0.
   // If not specified, use the first available device as default.
   uint32 id = 1;
+
+  // Control how many times Envoy makes an attempt to send event to DLB hardware.
+  // No retry as default.
+  uint32 retry_times = 2;
```

Comment:
nit: consider renaming to `max_post_retries`, or something that hints that this is for posting/sending an event.
Also consider adding a maximal value here, if there is something that makes sense.

Reply:
Since the value depends on specific cases, so hard to limit max here.